### PR TITLE
Test commit suggestions with minor fix.

### PR DESCRIPTION
**Pull request by wadepickett**
Testing commit suggestions with minor fix on clean PR.

### CODE REVIEW

**Comment by wadepickett** [comment]
@Rick-Anderson.  New clean PR to test commit suggestion on.  Thanks again for the help.

File: aspnetcore/razor-pages/razor-pages-conventions.md [hunk 1]
```diff
@@ -392,7 +392,7 @@ Visit the Contact page at either its ordinary route, `/Contact`, or the custom r
 
 ## Page model action conventions
 
-The default page model provider that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions which are designed to provide extensibility points for configuring page models. These conventions are useful when building and modifying page discovery and processing scenarios.
+The default page model provider that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions that are designed to provide extensibility points for configuring page models. These conventions are useful when building and modifying page discovery and processing scenarios.
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/razor-pages-conventions.md]: The page model provider that implements <xref:Microsoft.AspNetCore.Mvc.ApplicationModels.IPageApplicationModelProvider> invokes conventions that are designed to provide extensibility points for configuring page models. These conventions are us

**Comment by wadepickett** [comment]
Commit suggestion option was finally just before you closed. Thanks.

**Comment by wadepickett** [comment]
@Rick-Anderson, commit suggestion still grayed out after branch came back.  Sending you a fresh PR in a sec.

**Comment by wadepickett** [comment]
What we figured out:
If you are not an owner of a repo (which I am not, I only have write access) then you have to use your own fork for PR's.  By using your own fork, you get to allow commit suggestions and add suggestions to a batch commit to any of your PR's on that fork.  The trade off of having to use a fork is the extra steps involved in keeping that fork up to date and working with PR's.

**Pull request closed without merging**